### ISSUE TITLE
Interference with clipboard. 

### ISSUE THREAD

**Issue by Alacritous** [exaile/exaile#748]
Working on my computer. finished scanning my collection and I'm playing music. Noticed a problem with a few tracks and I'm trying to correct the error by copying and pasting from one track to another in Explorer and can't copy/paste in any app. Right click copy or cut doesn't do anything and pasting doesn't work. Other applications like Ditto and Event Ghost can't access the clipboard at all and generate errors. 

After much wailing and gnashing of teeth, restart exaile and the problem goes away. 

What the eff?  Why is Exaile futzing with the clipboard?   

### Steps to Reproduce (for bugs)

1.  No idea. Problem went away after closing exaile 

### Environment

* Operating System and version:

* Exaile Version:

**Comment by sjohannes** [comment]
Could you edit the report and add the Exaile version and OS? It would help us diagnose the problem.

**Comment by Alacritous** [comment]
Done. info added.

**Comment by sjohannes** [comment]
I've had this happen in the past but couldn't really figure it out. Do you have the "Multimedia keys for Windows" plugin enabled? That's the only thing I can think of that could interfere with other programs. Otherwise it could be just some random intermittent GTK bug.

**Comment by Alacritous** [comment]
Nope. no plugins except for the defaults. I installed it some time ago and only just remembered and was playing music and then I couldn't access the clipboard from anything. 

![image](https://user-images.githubusercontent.com/1443724/111566641-278d0f00-8763-11eb-97e7-46d6b3ae89a9.png)

**Comment by Alacritous** [comment]
Does the program touch the clipboard in any way whatsoever?

**Comment by sjohannes** [comment]
> Does the program touch the clipboard in any way whatsoever?

Only in two places: *File → Open URL* and *Radio (panel) → Add Station*. They read the clipboard and paste the content if it's a valid URL.

**Comment by phalgunlolur** [comment]
Oh wow. I have this happening to me from time to time. Never even thought it could be something related to Exaile. I always assumed it was a Windows issue and was planning on reinstalling my OS over the summer! Will keep an eye out on when it happens again and keep people posted here!

**Comment by sazawal** [comment]
I am still facing the issue on Exaile 4.1.3-beta2

**Comment by Chealer** [comment]
@Alacritous: Thanks for reporting, but please provide minimal steps to reproduce. Even if you do not know how to fully reproduce, list how it reproduces once “the problem has started”, and give an estimate of how many times it started.

**Comment by Chealer** [comment]
If you report experiencing this, please at least indicate your OS.
